src/reference/modules/administration/pages/procedures.adoc

```diff
@@ -159,6 +166,12 @@ tar -C /var/rudder -zcf rudder-backup-$(date +%Y%m%d).tar.gz configuration-repos
 # Then backup Rudder configuration
 tar -C /opt/rudder -zcf rudder-etc-backup-$(date +%Y%m%d).tar.gz etc/
 
+# Backup the apache2 Rudder configuration file
+tar -C /var/rudder -zcf rudder-apacahe2-backup-$(date +%Y%m%d).tar.gz /etc/apache2/sites-enabled/
+
+# Backup the certificates files present in /etc/apache2/sites-enabled/rudder.conf
```

Comment:
this one need more details, like
"read the file, find were the certificat files are, copy them